src/coreclr/vm/eventing/eventpipe/ep-rt-coreclr.h

```diff
@@ -1019,21 +1023,63 @@ void
 ep_rt_system_time_get (EventPipeSystemTime *system_time)
 {
 	STATIC_CONTRACT_NOTHROW;
+    
+#ifdef HOST_WINDOWS
```

Comment:
This is duplicating the code from aot EventPipe. They are the only users of system time now and wrapping through minipal can introduce unnecessary complexity. Maybe we can share code among EventPipe implementations as many functions are moving to minipal?